src/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -72,6 +76,7 @@ public abstract partial class PipeWriter : System.Buffers.IBufferWriter<byte>
     public readonly partial struct ReadResult
     {
         private readonly object _dummy;
+        private readonly int _dummyPrimitive;
```

Comment:
What's the _dummyPrimitive for, and why do we need _dummy too?

Reply:
Autogenerated. http://blog.paranoidcoding.com/2016/02/15/are-private-members-api-surface.html

Reply:
> Autogenerated.

woot :)